MyPortfolio.services.api/Program.cs

```diff
@@ -0,0 +1,22 @@
+// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace MyPortfolio.Services.api
+{
+    public class Program
+    {
+        public static void Main(string[] args) =>
+         CreateHostBuilder(args).Build().Run();
```

Comment:
Fix indentation here and other places too.

Reply:
Fixed in both Program.cs and Startup.cs